src/sidebar/components/group-list-item.js

```diff
@@ -7,9 +7,14 @@ const { createElement } = require('preact');
 const { orgName } = require('../util/group-list-item-common');
 const { withServices } = require('../util/service-context');
 
-function GroupListItem({ analytics, group, store }) {
+function GroupListItem({ analytics, group, settings, store }) {
   const focusGroup = () => {
     analytics.track(analytics.events.GROUP_SWITCH);
+
+    // Clear the direct-linked annotations and group from settings.
+    delete settings.annotations;
```

Comment:
I wanted to keep the settings as they would have appeared had these settings not been set and in those cases the property is literally missing.